libs/blocks/youtube/youtube.js

```diff
@@ -14,12 +14,18 @@ export default function init(a) {
         webkitallowfullscreen mozallowfullscreen allowfullscreen
         allow="encrypted-media; accelerometer; gyroscope; picture-in-picture"
         scrolling="no"
+        id='player-${id}'
```

Comment:
Nit: other attribute values are surrounded by double quotes, this is surrounded by single quotes, we should follow the same pattern here as well.

Reply:
+1. this is HTML. Should be double quotes.

Reply:
done